src/_guides/language/language-tour.md

```diff
@@ -1398,8 +1398,8 @@ assert(say('Bob', 'Howdy', 'smoke signal') ==
 <a id="default-parameters"></a>
 #### Default parameter values
 
-Your function can use `=` to define default values for both named and positional
-parameters. The default values must be compile-time constants.
+Your function can use `=` to define default values for both named and optional
+positional parameters. The default values must be compile-time constants.
```

Comment:
What are your thoughts on something like the following? Since technically required named arguments [cannot have a default value defined](https://dart.dev/tools/diagnostic-messages#default_value_on_required_parameter) either. Happy to consider other wordings as well. 

```suggestion
Your function can use `=` to define default values for optional parameters,
both named and positional. The default values must be compile-time constants.
```

Reply:
Thanks for the complement, @parlough. The description is clear enough now.